escposprinter/src/main/java/com/dantsu/escposprinter/EscPosPrinter.java

```diff
@@ -156,6 +156,7 @@ public EscPosPrinter printFormattedTextAndCut(String text, int dotsFeedPaper) th
 
         this.printFormattedText(text, dotsFeedPaper);
         this.printer.cutPaper();
+        this.printer.openCashBox();
```

Comment:
Remove this.
Add a new method `printFormattedTextAndOpenCashBox(String text, int dotsFeedPaper)` like this : 

```java
    /**
     * Print a formatted text, cut the paper and open the cash box. Read the README.md for more information about text formatting options.
     *
     * @param text Formatted text to be printed.
     * @param dotsFeedPaper distance feed paper at the end.
     * @return Fluent interface
     */
    public EscPosPrinter printFormattedTextAndOpenCashBox(String text, int dotsFeedPaper) throws EscPosConnectionException, EscPosParserException, EscPosEncodingException, EscPosBarcodeException {
        if (this.printer == null || this.printerNbrCharactersPerLine == 0) {
            return this;
        }

        this.printFormattedTextAndCut(text, dotsFeedPaper);
        this.printer.openCashBox();
        return this;
    }
```

Reply:
ok,good idea!